app.js

```diff
@@ -1,8 +1,11 @@
-const express = require("express")
-const app = express()
-const { getApi } = require("./api.controller")
+const express = require("express");
+const app = express();
+const { getApi, getTopics } = require("./api-controller");
 
+app.use(express.json());
```

Comment:
Are we using this line of code yet? If not, it should be removed until we need it.